api/routers/report.js

```diff
@@ -4,6 +4,6 @@ const MainController = require('../controllers/main');
 const { csrfProtection } = require('../middlewares');
 
 // add csrf middleware to app route so that we can use request.csrfToken()
-router.get('/report(/*)?', csrfProtection, MainController.report);
+router.get('/report{*splat}', csrfProtection, MainController.report);
```

Comment:
Should this be `{/*splat}` with the forward slash like the `routes/main.js` changes?